modules/rostests/apitests/gdi32/SetWorldTransform.c

```diff
@@ -70,15 +71,15 @@ void Test_SetWorldTransform()
     result = GetWorldTransform(hdc, &xform);
     ok(result == 1, "GetWorldTransform should succeed\n");
     ok(xform.eM11 == 2, "xform.eM11 should be 2\n");
-    ok(xform.eM12 == (FLOAT)3.0001, "xform.eM11 should be 3.0001\n");
-    ok(xform.eM21 == 4, "xform.eM11 should be 4\n");
-    ok(xform.eM22 == 6, "xform.eM11 should be 6\n");
+    ok(xform.eM12 == (FLOAT)3.0001, "xform.eM12 should be 3.0001\n");
+    ok(xform.eM21 == 4, "xform.eM21 should be 4\n");
+    ok(xform.eM22 == 6, "xform.eM22 should be 6\n");
 
     /* Set smallest possible values */
-    xform.eM11 = (FLOAT)1.4e-45;
+    xform.eM11 = 1.17549435e-38f;
     xform.eM12 = 0;
     xform.eM21 = 0;
-    xform.eM22 = (FLOAT)1.4e-45;
+    xform.eM22 = 1.17549435e-38f;
```

Comment:
What happens if one uses the older values?

Reply:
Fails in Win2k3.